livingatlas/pipelines/pom.xml

```diff
@@ -0,0 +1,890 @@
+        <extensions>true</extensions>
+        <executions>
+          <execution>
+            <id>start</id>
+            <phase>pre-integration-test</phase>
+            <goals>
+              <goal>start</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>stop</id>
+            <phase>post-integration-test</phase>
+            <goals>
+              <goal>stop</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <images>
+            <image>
+              <alias>solr8</alias>
+              <name>djtfmartin/biocache-pipelines-solr8:v3</name>
+              <external>
+                <type>compose</type>
+                <composeFile>solr8.yml</composeFile>
+              </external>
+            </image>
+            <image>
+              <alias>ala-nameservice</alias>
+              <name>djtfmartin/ala-namematching-service:v20200214-4</name>
+              <external>
+                <type>compose</type>
+                <composeFile>ala-nameservice.yml</composeFile>
+              </external>
+              <run>
+                <wait>
+                  <http>
+                    <url>http://localhost:9179/api/search?q=Acacia</url>
+                    <method>GET</method>
+                    <status>200</status>
+                  </http>
+                  <time>20000</time>
+                </wait>
+              </run>
+            </image>
+          </images>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <properties>
+    <!-- Common variables -->
+    <jdkLevel>1.8</jdkLevel>
+    <encoding>UTF-8</encoding>
+    <surefireArgLine>-Xmx1024m</surefireArgLine>
+
+    <project.build.sourceEncoding>${encoding}</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>${encoding}</project.reporting.outputEncoding>
+    <project.resources.sourceEncoding>${encoding}</project.resources.sourceEncoding>
+
+    <!-- Tools-->
+    <gbif-pipelines.version>2.5.4-SNAPSHOT</gbif-pipelines.version>
+    <geotools.version>20.5</geotools.version>
+    <jackson.version>[2.9.10.3,2.11)</jackson.version>
+    <commons-configuration.version>1.6</commons-configuration.version>
+    <netty.version>3.9.9.Final</netty.version>
+    <netty-all.version>4.1.42.Final</netty-all.version>
+    <ingest-hdfs-table.version>2.2.34</ingest-hdfs-table.version>
+    <snakeyaml.version>1.25</snakeyaml.version>
+    <streamex.version>0.7.2</streamex.version>
+    <!-- This is overriding GBIF mother pom which is using a 2.9 version which causes issues -->
+    <jackson-module-scala_2.11.version>2.11.0</jackson-module-scala_2.11.version>
+    <jcommander.version>1.78</jcommander.version>
+    <layers-store.version>2.0.3</layers-store.version>
+
+    <!-- Plugins -->
+    <io.fabric8.version>0.33.0</io.fabric8.version>
+    <org.jetbrains.annotations>13.0</org.jetbrains.annotations>
+
+  </properties>
+
+  <scm>
```

Comment:
This looks suspicious

Reply:
i presume this was the jackson version which was using the range. This has been changed.